entity-web/src/main/java/eu/europeana/entity/web/controller/BaseRest.java

```diff
@@ -115,6 +115,14 @@ protected String serializeMetricView(EntityMetric metricData) throws EntityApiRu
         return jsonLdSerializer.serializeToJson(metricData);
     }
 
+    /**
+     * If apikey service url is empty, disable the authentication
+     * @return
+     */
+    protected boolean isAuthEnabled() {
```

Comment:
as we need this for allmost all APIs, it will be good to move the implementation to api-commons BaseRestController